dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/common/DefaultIdentifiableObjectManager.java

```diff
@@ -943,11 +871,11 @@ public <T extends IdentifiableObject> List<T> getObjects( Class<T> type, Identif
             {
                 return store.getByUid( identifiers );
             }
-            else if ( IdentifiableProperty.CODE.equals( property ) )
+            if ( IdentifiableProperty.CODE.equals( property ) )
```

Comment:
since this is an enum should we move to `==` instead of `equals`?
just because you touched this line, I know its not part of the scope of your PR. so you can also leave it 😋